lib/gollum/views/page.rb

```diff
@@ -184,6 +192,42 @@ def content_without_page_header(content)
         # .inner_html will cause href escaping on UTF-8
         doc.css("div#gollum-root").children.to_xml(@@to_xml)
       end
+
+      def table(data)
+        result = "<table>\n"
+        case data
+          when Hash
```

Comment:
They do handle `Array`s judging from the screenshot here: https://github.com/blog/1647-viewing-yaml-metadata-in-your-documents

Can we come up with a way to do this without explicitly checking for class?